Nos/Views/AuthorStoryView.swift

```diff
@@ -10,14 +10,20 @@ import CoreData
 
 struct AuthorStoryView: View {
     
-    @ObservedObject var author: Author
+    var author: Author
```

Comment:
we not need to make it observed as the parent view displays all authors in a tabview instead of just displaying the selected author